extension/popup.js

```diff
@@ -101,6 +101,114 @@ const downloadStatusIndicator = document.getElementById(
   "downloadStatusIndicator"
 );
 
+
+// DOM elements (theme)
+const lightTheme = document.getElementById("lightTheme");
```

Comment:
Instead of manipulating every style property via JS, define CSS classes for .light-theme and .dark-theme in your stylesheet. Then, toggle the appropriate class on a parent element (such as or .container).